discretisedfield/mesh.py

```diff
@@ -1,4 +1,5 @@
 import k3d
```

Comment:
imports could be sorted better: internal packages first, then external (such as k3d), then our own). Within each category, ideally in alphabetical order.

`isort` will check/correct this I think.

Reply:
Will be done in a separate PR